File: python_solve/baek1929.py
import sys
import math
input = sys.stdin.readline

#prime_nums = []
# def is_prime(num):
#     if num in prime_nums:
#         print(num)
#         return
#     else:
#         sqrtn = int(math.sqrt(num))
#         for i in range(2, (sqrtn+1)):
#             if (num % i) == 0:
#                 return False
#         prime_nums.append(num)
#         print(num)
#         return
def is_prime(num):
    sqrtn = int(math.sqrt(num))
    for i in range(2, (sqrtn+1)):
        if (num % i) == 0:
            return False
    return True

M, N = map(int, input().split())
for i in range(M, N+1):
    if i == 1:
        continue
    if is_prime(i):
        print(i)

File: java_solve/Baek1929.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Baek1929 {

public static void main(String[] args) throws Exception {
    BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));
    StringTokenizer st = new StringTokenizer(bf.readLine());
    int M = Integer.parseInt(st.nextToken());
    int N = Integer.parseInt(st.nextToken());
    for(int i=M; i<=N; i++){
        isPrime(i);
    }

}
private static void isPrime(int num){
    if(num==1){
        return;
    }
    for(int i=2; i<=Math.sqrt(num); i++){
        if((num%i) == 0){
            return;
        }
    }
    System.out.println(num);

}

}
